.changes/next-release/Feature-ece686c4-fb2c-4773-ac2e-41b33b234b0f.json

```diff
@@ -0,0 +1,4 @@
+{
+    "type": "Feature",
+    "description": "Import icons from vscode-icons"
```

Comment:
Since these icons aren't replacing any existing icons, there is no user facing change. This changelog entry can be removed

Reply:
done